satellite/orders/endpoint.go

```diff
@@ -208,7 +209,11 @@ func (endpoint *Endpoint) Settlement(stream pb.Orders_SettlementServer) (err err
 		now := time.Now().UTC()
 		intervalStart := time.Date(now.Year(), now.Month(), now.Day(), now.Hour(), 0, 0, 0, now.Location())
 
-		if err := endpoint.DB.UpdateBucketBandwidthSettle(ctx, bucketID, orderLimit.Action, order.Amount, intervalStart); err != nil {
+		projectID, bucketName, err := SplitBucketID(bucketID)
```

Comment:
UseSerialNumber should be returning project id and bucket name separately.

Reply:
the serial number tables stores the bucketID so thats what it returns. I would have to split the bucketID

Reply:
ah, right... I thought we already had split that as well